Contrasts schema validation (#410)
nf-core/differentialabundance: docs/usage.md

## Contrasts file

> The contrasts file references the observations file to define groups of samples to compare. It can be provided in both CSV/TSV and YAML format using the parameters `--contrasts` or `--contrasts_yml`, respectively. Please note that only one of these parameters can be used at the same time.

Reviewer: ```suggestion
The contrasts file references the observations file to define groups of samples to compare. It can be provided in **either** CSV/TSV or YAML format using the parameters `--contrasts` or `--contrasts_yml`, respectively. 
```

### CSV contrasts file

```bash
--contrasts '[path to CSV contrasts file]'
```

> Based on the sample sheet above we could define contrasts like:

Reviewer: ```suggestion
The contrasts file references the observations file to define groups of samples to compare. For example, based on the sample sheet above we could define contrasts like:
```

Sorry, I liked that additional bit of doc
